apps/app/webapp.tf

```diff
@@ -216,3 +216,19 @@ resource "azuread_group_member" "developers" {
   group_object_id  = var.developers_ad_group_principal_id
   member_object_id = azuread_group.developers_of_this_app.id
 }
+
+resource "azurerm_cosmosdb_sql_role_assignment" "data_scientists" {
```

Comment:
Also suggest you move this to the `cosmos.tf` file as it pertains to cosmos and not the web app